### PR TITLE
Handle default arguments and argument number to avoid errors

### PR DESCRIPTION
**Pull request by carrieedwards**
This PR adds error handling to function arguments. If fewer arguments than the minimum required by that function are passed in, then a missing argument error is now returned. Additionally, some functions with default values for arguments were missing the default, so those have been updated. This will prevented unexpected out-of-index panics and other errors from occurring.

### CODE REVIEW

File: expr/functions/aggregateLine/function.go [hunk 1]
```diff
@@ -37,24 +37,13 @@ func (f *aggregateLine) Do(ctx context.Context, e parser.Expr, from, until int64
...
-		callback, err = e.GetStringArg(1)
-		if err != nil {
-			return nil, err
-		}
-
-		keepStep, err = e.GetBoolArgDefault(2, false)
-		if err != nil {
-			return nil, err
-		}
+	callback, err := e.GetStringArgDefault(1, "average")
```
Comment (ywwg) [inline expr/functions/aggregateLine/function.go:40]: what is the story behind this part of the change?

File: expr/functions/highestLowest/function.go [hunk 2]
```diff
@@ -68,18 +71,14 @@ func (f *highest) Do(ctx context.Context, e parser.Expr, from, until int64, valu
 			if err != nil {
 				// We need to support case where only function specified
 				n = 1
-				consolidation, err = e.GetStringArg(1)
-				if err != nil {
-					return nil, err
-				}
```
Comment (npazosmendez) [inline expr/functions/highestLowest/function.go:74]: Why was this deleted?

File: expr/functions/randomWalk/function.go [hunk 3]
```diff
@@ -28,7 +28,7 @@ func New(configFile string) []interfaces.FunctionMetadata {
 	return res
 }
 
-// squareRoot(seriesList)
+// randomWalk(seriesList, step=60)
```
Comment (npazosmendez) [inline expr/functions/randomWalk/function.go:31]: nit: this hould be `randomWalk(name, step=60)`

File: expr/functions/aggregateLine/function.go (hunk 1 above)
Comment (carrieedwards) [reply expr/functions/aggregateLine/function.go:40]: I just replaced the switch statement with functions that will set the default argument

**Comment by carrieedwards** [comment]
Closing in favor of https://github.com/go-graphite/carbonapi/pull/712
